Dockerfile

```
@@ -1,16 +1,15 @@
-FROM golang:1.17-alpine as builder
-RUN go install go.k6.io/xk6/cmd/xk6@latest
-RUN apk --no-cache add git
+FROM golang:1.16-alpine as builder
 WORKDIR $GOPATH/src/go.k6.io/k6
 ADD . .
-RUN xk6 build --output /go/bin/k6 \
-	--with github.com/grafana/xk6-output-influxdb=.
+RUN apk --no-cache add git
+RUN CGO_ENABLED=0 go install go.k6.io/xk6/cmd/xk6@latest
+RUN CGO_ENABLED=0 xk6 build --with github.com/grafana/xk6-output-influxdb=. --output /tmp/k6
```

```suggestion
RUN CGO_ENABLED=0 xk6 build --with github.com/grafana/xk6-output-influxdb=.
```
Why not use the default folder?

---

@codebien  -> Line 11 -> `COPY --from=builder /tmp/k6 /usr/bin/k6`

---

it is the motivation why the Line 11 before was `COPY --from=builder /go/bin/k6 /usr/bin/k6` for using the default Go binary folder